src/web/components/MostViewedList/MostViewedList.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { useApi } from '@root/src/web/components/lib/api';
+import { GuardianLines } from '@root/src/web/components/GuardianLines';
+import { css } from 'emotion';
+import { textSans } from '@guardian/src-foundations/typography';
+import { MostViewedListItem } from './MostViewedListItem';
+
+const wrapperStyles = css`
+    margin-top: 24px;
+`;
+
+const headingStyles = css`
+    ${textSans.medium({ fontWeight: 'bold' })}
+    margin-bottom: 8px;
+`;
+
+export interface TrailType {
+    url: string;
+    linkText: string;
+    isLiveBlog: boolean;
+    ageWarning: string;
+    image?: string;
+    pillar: Pillar;
+}
+
+export interface TabType {
+    heading: string;
+    trails: TrailType[];
+}
+
+interface Props {
+    limitItems?: number;
+}
+
+export const MostViewedList = ({ limitItems = 5 }: Props) => {
+    const endpointUrl: string =
+        'https://api.nextgen.guardianapps.co.uk/most-read-geo.json?dcr=true';
+    const { data, error } = useApi<any>(endpointUrl);
```

Comment:
Is there only 1 variation of this endpoint? I have no reason to think otherwise but just wanted to be sure as these lists of articles are often very context sensitive

Reply:
Yes, I've confirmed this is the endpoint where the data should always be fetched from.